content/en/docs/reference/glossary/primary-cluster.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Primary Cluster
+test: n/a
+---
+
+A primary cluster is a cluster with a control plane, which observes all
+clusters in the mesh for endpoints and facilitates cross-cluster load balancing
```

Comment:
Is this correct?
```suggestion
A primary cluster is a cluster with a control plane, which observes more than one
cluster in a multicluter mesh for endpoints and facilitates cross-cluster load balancing
```

Reply:
@nmittler @smawson WDYT?

Reply:
I think the original text was correct. A single-cluster mesh consists of a single primary cluster.

Reply:
I was worried about the other end, "observes all clusters". Is that always true? Isn't it possible to have a primary cluster that only observes a subset of the clusters in the mesh, while another primary cluster handles others?

Reply:
Not in any documented configuration that I'm aware of. Unless you're talking about versioned control planes, which is slightly orthogonal.

Reply:
@frankbu is correct. A single istiod control plane can be configured to observe only a subset of a cluster.

Also the statement “which observes more than one cluster” is too strict. Is should be “which observes resources from one or more clusters...”.

Reply:
@dcberg 

> @frankbu is correct. A single istiod control plane can be configured to observe only a subset of a cluster.

Just to be clear, are you referring to control plane revisions?

If we want to be more precise, maybe we can rephrase to something like:

```
A primary cluster is a cluster within a mesh that contains a control plane. The control plane observes configuration resources from a single source (e.g. Kubernetes API Server). It also observes service endpoints from one or more clusters in the mesh in order to facilitate cross-cluster load balancing between workloads.
```

Reply:
> Multiple primary clusters can be connected together

I don't think what "connected together" means is clear. What's it trying to say? You can have more than one (synchronized) primary cluster in the same mesh for HA? I think it needs to say something more specific.